### PR TITLE
Adds a filter for youtube embeds

### PR DESCRIPTION
**Pull request by gboone**
This will make it easier for editing in federalist's editor eventually. In the meantime it will make sure all youtube embeds are properly scaled at smaller screen sizes. cc: @18F/blog

### CODE REVIEW

File: assets/_sass/components/_embeds.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .embed-container {
   position: relative;
-  padding-bottom: 100%;
+  padding-bottom: 56.25%;
```
Comment (rogeruiz) [inline assets/_sass/components/_embeds.scss]: Would there also need to be a `padding-top: 25px;` to allow for the YouTube chrome on the embedded player?

File: _plugins/embeds.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module Jekyll
+  module EmbedFilter
+    # Pass a url like {{ "https://youtube-nocookie.com/blahblahblah/" | embed }}
+    # It will spit out a standardized embed
+    def embed(input)
+      "<div class='embed-container'><iframe src='#{input}' width='560' height='315' frameborder='0' allowfullscreen></iframe></div>"
```
Comment (gboone) [inline _plugins/embeds.rb]: OK ignoring this because it's an HTML string and the length doesn't impede readability.

**Comment by rogeruiz** [comment]
Looks good to merge in @gboone :+1:

File: assets/_sass/components/_embeds.scss (hunk 1 above)
Comment (gboone) [inline assets/_sass/components/_embeds.scss]: Good idea!

**Comment by gboone** [comment]
thanks @rogeruiz!

**Pull request merged**